CONTRIBUTION.md

```diff
@@ -0,0 +1,81 @@
+# How to contribute
+
+Windows Machine Config Operator is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This
+document outlines some of the conventions on commit message formatting and other resources to help get contributions into the project.  
+
+
+## Reporting bugs and creating issues
+
+If any part of the project has bugs or documentation mistakes, please let us know by opening a
+[Jira issue](https://jira.coreos.com/projects/WINC/summary) (Red Hat internal) or a
+[GitHub issue](https://github.com/openshift/windows-machine-config-operator/issues/new) (external users) or a PR.
```

Comment:
Are `external users` still needed to open github issues? I think they've access to jira?